src/models/index.ts

```diff
@@ -0,0 +1,14 @@
+export {default as User} from './user';
+
+export abstract class AccessToken {
+    public abstract id: number;
+    public abstract token: string;
+
+    public revoke(): Promise<void> {
+        return new Promise((): void => {});
+    }
+}
+
+export abstract class RefreshToken {
+    public abstract id: number;
```

Comment:
don't we have user id and token here as well?

Reply:
@hesamd  This is out of the scope of this PR, this class here is just a mock, it'll be implemented in another PR.